test/failure_app_test.rb

```diff
@@ -294,5 +294,22 @@ def call_failure(env_params={})
       assert @response.third.body.include?('<h2>Log in</h2>')
       assert @response.third.body.include?('Your account is not activated yet.')
     end
+
+    if Rails.application.config.respond_to?(:relative_url_root)
```

Comment:
I was under the impression `relative_url_root` could be left blank in the application config.